Documentation/flexvolume.md

```diff
@@ -60,6 +60,10 @@ The kubelet on GKE is already configured to use that directory.
 
 Continue with [configuring the FlexVolume path](#configuring-the-flexvolume-path) to configure Rook to use the FlexVolume path.
 
+### Kubespray
+Kubespray defaults to a read/write FlexVolume path of `/var/lib/kubelet/volume-plugins`.
+You will need to use this value when [configuring the Rook operator](#configuring-the-rook-operator)
```

Comment:
Could you please form this section as the other are done, so it is all in mostly the same form.

```
Kubespray uses a non-standard FlexVolume plugin directory: `/var/lib/kubelet/volume-plugins`.
The Kubespray configured kubelet is already configured to use that directory.

Continue with [configuring the FlexVolume path](#configuring-the-flexvolume-path) to configure Rook to use the FlexVolume path.
```

Reply:
Okay I made the suggested changes and added link to where it's set in case they change the default.